ci/README.md

```diff
@@ -0,0 +1,40 @@
+# Continuous Integration with GitHub Actions
+
+[Documentation about GitHub Actions](https://docs.github.com/en/actions)
+
+## Builds
+
+Builds will fail if any command has a non-zero exit code.
+PowerShell scripts continue on non-terminating errors unless the file is prefixed with `$ErrorActionPreference = "Stop";`.
```

Comment:
I think we should also consider `Set-Strictmode -Version latest`, which avoids the case where nonexistent variables default to `$null`.

Reply:
what happens when there are non-existent variables? I think we rely on non existing variables being none - see `feature_` flags for example

Reply:
In that case, let's leave this as is.
I use PowerShell in my daily job and there we always enforce strict mode, since that forces us to think about these cases and initialize the variables first. Without Strict Mode, PowerShell is not typo proof per se. If it works, don't break it though.